encord/client.py

```diff
@@ -1065,5 +1074,82 @@ def __set_project_ontology(self, ontology: Ontology) -> bool:
         payload = {"editor": ontology.to_dict()}
         return self._querier.basic_setter(Project, uid=None, payload=payload)
 
+    def get_project_labels_consensus(
+        self,
+        comparing_projects: List[EncordClientProject],
+        ontology_feature_node_hash_list: List[str],
```

Comment:
I remember it was my suggestion, but if you keep this `ontology_feature_node_hash` (which will be nicer to read for the public interface) and then do 

`        feature_node_hash_set = set(ontology_feature_node_hash_list)` below I think it would be the right balance of having a nice interface and also code clarity further down the line.

Also, is this a `List` because we usually return a list of ontology feature node hashes?

Reply:
I used `List` because `client.get_project_ontology().ontology_objects` returns a list of `OntologyObject` and the feature node hashes are extracted from this list but as they are extracted they could be stored in a set too.
`[obj.feature_node_hash for obj in client.get_project_ontology().ontology_objects]`
vs
`{obj.feature_node_hash for obj in client.get_project_ontology().ontology_objects}`
At the end, there's no strong criteria about any choice as the user doesn't need to worry about that in order to solve the task at hand. We could use `Iterable` but I think that choice gives the wrong idea as the method stores in memory all the data at since the beginning.